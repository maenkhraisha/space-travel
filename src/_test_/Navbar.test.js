import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter as Router } from 'react-router-dom';
import NavLink from '../Components/Navbar';

it('renders correctly', () => {
  const tree = renderer
    .create(
      <Router>
        <NavLink />
      </Router>,
    ).toJSON();
  expect(tree).toMatchSnapshot();
});
