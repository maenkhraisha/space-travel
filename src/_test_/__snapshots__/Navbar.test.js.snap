// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<nav
  className="navbar"
>
  <div
    className="nav-img"
  >
    <img
      alt="Logo"
      src="Photo.png"
      style={
        Object {
          "height": "100px",
          "width": "100px",
        }
      }
    />
    <h1>
      Space Travel Hub
    </h1>
  </div>
  <div
    className="nav-links"
  >
    <a
      className="link"
      href="/rockets"
      onClick={[Function]}
    >
      Rockects
    </a>
    <a
      className="link"
      href="/missions"
      onClick={[Function]}
    >
      Missions
    </a>
    <a
      className="link"
      href="/profile"
      onClick={[Function]}
    >
      My Profile
    </a>
  </div>
</nav>
`;
